### ISSUE TITLE
[feat] Support help text for fields

### ISSUE THREAD

**Issue by u0816** [palmtreephp/form#15]
Sometimes, you need to provide contextual information to ensure accurate input.
```php
$builder->add('new_password', PasswordType::class, [
 'label' => 'New password',
 'help_text' => 'Your password must be 8-20 characters long, contain letters and numbers, and must not contain spaces, special characters, or emoji.'
]);
```

The help text is always rendered. Frontend developers can decide how to handle the error case.
```diff
// class AbstractType
public function getElements(): array
{
    $elements = [];

    $label = $this->getLabelElement();

    if ($label instanceof Element) {
        $elements[] = $label;
    }

    $element = $this->getElement();

    if (!isset($element->attributes['id'])) {
        $element->attributes['id'] = $this->getIdAttribute();
    }

    $isValid = $this->isValid();

    if (!$isValid) {
        $element->classes[] = 'is-invalid';
    }

    $elements[] = $element;

    
+   if (!empty($this->helpText)) {
+      $elementHelpText = Element::create('div.helptext.form-text')->setInnerText($this->helpText);
+      if (!$isValid) {
+          $elementHelpText->classes[] = 'is-invalid';
+      }
+      $elements[] = $elementHelpText;
+   }

   if (!$isValid && $this->errorMessage) {
        $elements[] = $this->form->createInvalidElement()->setInnerText($this->errorMessage);
    }

    return $elements;
}


+ protected $helpText = '';
+ public function setHelpText(string $helpText): TypeInterface
+ { $this->helpText = $helpText; return $this; }
+ public function getHelpText(): ?string
+ { return $this->helpText; }
```

**Comment by andyexeter** [comment]
Hey there,

Thanks for this. I was sure that this was already built in but I guess not! I'll see if I can implement it over the weekend.

I'd be happy to accept a PR if you wanted to do it yourself :)

**Comment by andyexeter** [comment]
This has been shipped in v4.5.1 - I went with `help` for the property name instead of `help_text`, e.g:

```php
$form->add('email', EmailType::class, [
    'help' => "We'll never share your email with anyone."
]);
```
